### PR TITLE
*: add link: deploy dm in k8s

### PR DESCRIPTION
**Pull request by Ehco1996**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-dm) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)

add `deploy dm in k8s` link to toc

### Which DM version(s) do your changes apply to? (Required)

- [x]master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)
- [x] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- Other reference link(s):
- https://docs.pingcap.com/zh/tidb-in-kubernetes/dev/deploy-tidb-dm

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: en/TOC.md [hunk 1]
```diff
@@ -26,6 +26,7 @@
     - [Use TiUP (Recommended)](deploy-a-dm-cluster-using-tiup.md)
     - [Use TiUP Offline](deploy-a-dm-cluster-using-tiup-offline.md)
     - [Use Binary](deploy-a-dm-cluster-using-binary.md)
+    - [Use kubernetes (experimental)](https://docs.pingcap.com/tidb-in-kubernetes/dev/deploy-tidb-dm)
```
Comment (CharLotteiu) [inline en/TOC.md:29]:
```suggestion
    - [Use Kubernetes (Experimental)](https://docs.pingcap.com/tidb-in-kubernetes/dev/deploy-tidb-dm)
```

File: zh/TOC.md [hunk 2]
```diff
@@ -31,6 +31,7 @@
     - [使用 TiUP（推荐）](deploy-a-dm-cluster-using-tiup.md)
     - [使用 TiUP 离线镜像](deploy-a-dm-cluster-using-tiup-offline.md)
     - [使用 Binary](deploy-a-dm-cluster-using-binary.md)
+    - [使用 kubernetes (实验性)](https://docs.pingcap.com/zh/tidb-in-kubernetes/dev/deploy-tidb-dm)
```
Comment (CharLotteiu) [inline zh/TOC.md:34]:
```suggestion
    - [使用 kubernetes (实验特性)](https://docs.pingcap.com/zh/tidb-in-kubernetes/dev/deploy-tidb-dm)
```

**Review by CharLotteiu — COMMENTED** [review]
rest LGTM

File: zh/TOC.md [hunk 3]
```diff
@@ -31,6 +31,7 @@
     - [使用 TiUP（推荐）](deploy-a-dm-cluster-using-tiup.md)
     - [使用 TiUP 离线镜像](deploy-a-dm-cluster-using-tiup-offline.md)
     - [使用 Binary](deploy-a-dm-cluster-using-binary.md)
+    - [使用 kubernetes (实验特性)](https://docs.pingcap.com/zh/tidb-in-kubernetes/dev/deploy-tidb-dm)
```
Comment (CharLotteiu) [inline zh/TOC.md:34]:
```suggestion
    - [使用 Kubernetes (实验特性)](https://docs.pingcap.com/zh/tidb-in-kubernetes/dev/deploy-tidb-dm)
```
Sorry, one more suggested change 😂

**Comment by CharLotteiu** [comment]
rest /lgtm

File: zh/TOC.md (hunk 3 above)
Comment (Ehco1996) [reply zh/TOC.md]: fixed

**Pull request merged**